### PR TITLE
More tweaks to GCP setup instructions

### PR DESCRIPTION
**Pull request by deflaux**
also

* now opening all external links in a new window
* silenced sphinx warnings about include files

The Bioc workshop material is still a work-in-progress so feel free to just skim that for now.

### CODE REVIEW

File: docs/source/includes/r_workbench_setup.rst [hunk 1]
```diff
@@ -1,7 +1,10 @@
-(1) Download the `BiocDockerOnGCE launch script`_ to your local machine.
 .. sidebar:: Details
 
-     This script will create virtual machine on Google Cloud Platform with a locked down network (only port 22 open).  Your local machine will securely connect to the VM via an ssh tunnel.
+  This script will create virtual machine on Google Cloud Platform with a locked down network (only port 22 open).  Your local machine will securely connect to the VM via an ssh tunnel.
```
Comment (mbookman) [inline docs/source/includes/r_workbench_setup.rst]: Maybe also note that port 22 is SSH

File: docs/source/workshops/bioc-2015.rst [hunk 2]
```diff
@@ -1,18 +1,13 @@
 BioC 2015: Where Software and Biology Connect
 =============================================
 
-Give an overview here ... *in this workshop you will* ...
+Google has some pretty amazing big data computational "hammers" that they have been applying to search and video data for a long time.  In this workshop we take those same hammers and apply them to whole genome sequences.
```
Comment (mbookman) [inline docs/source/workshops/bioc-2015.rst]: Unless we can attribute this language to someone not at Google, we might want to remove some of the hyperbole (whether valid or not).

**Comment by mbookman** [comment]
Lots of very good stuff in here. Adding the screenshots will really help users.
Two small comments. Otherwise LGTM.

**Comment by pgrosu** [comment]
I concur with Matt, the screenshots help a lot!  I always try to include some in my docs (picture is worth ... :)

Thanks for the nice work!
~p

**Comment by deflaux** [comment]
thanks @mbookman! I'm about to start on a bunch of other feedback so I'll add port 22 in the next round.

**Pull request merged**